app/services/bulk_ingest_service.rb

```diff
@@ -221,6 +220,17 @@ def to_a
         nodes
       end
 
+      def mime_type(file_path)
+        basename = File.basename(file_path)
```

Comment:
do we need this line? It looks like we do this right before calling the method, and don't use file_path anywhere else in this method. just accept basename?